pkg/core/network.go

```diff
@@ -497,6 +522,7 @@ func (n *NetworkCommand) PartitionChain(ipset string) ([]*pb.Chain, error) {
 		Name:      fmt.Sprintf("%s/1", directionStr),
 		Ipsets:    []string{ipset},
 		Direction: directionChain,
+		Protocol:  n.IPProtocol,
```

Comment:
I think protocol maybe not meaningful here , since if one of input & output chain drop , (tcp,udp,icmp) will all broken.

Reply:
`iptables` can drop network package by specified protocol, the command looks like `iptables -w -A OUTPUT/1 -m set --match-set chaos-962d44ca-a0dc-4f dst -j DROP -w 5 --protocol tcp`, so I think the `protocol` here is meaningful

Reply:
Sorry , I misunderstand it. I agree with you.